### PR TITLE
Add a GCN download button

### PR DESCRIPTION
**Pull request by mcoughlin**
This PR adds a GCN download button to the observation table.

### CODE REVIEW

File: skyportal/handlers/api/observation.py [hunk 1]
```diff
@@ -613,3 +674,146 @@ def post(self):
...
+            return self.error(message=f"Invalid instrument ID {instrument_id}")
+
+        print(
+            start_date,
+            end_date,
+            localization_dateobs,
+            localization_name,
+            localization_cumprob,
+            return_statistics,
+        )
```
Comment (guynir42) [inline skyportal/handlers/api/observation.py:776]: Do we need this print statement?

File: skyportal/tests/flaky/test_frontend_flaky.py [hunk 2]
```diff
@@ -355,3 +355,108 @@ def test_gcnevents_observations(
...
+        'observationData': pd.read_csv(datafile).to_dict(orient='list'),
+    }
+
+    status, data = api('POST', 'observation', data=data, token=super_admin_token)
+
+    assert status == 200
+    assert data['status'] == 'success'
+
+    # wait for the executed observations to populate
+    time.sleep(15)
```
Comment (guynir42) [inline skyportal/tests/flaky/test_frontend_flaky.py:429]: We can definitely do better without this `sleep`. First let the driver get `become_user` and the event page, and after that there's a `wait_for_xpath` anyway.

**Review by guynir42 — APPROVED** [review]
Minor comments.

File: skyportal/handlers/api/observation.py (hunk 1 above)
Comment (mcoughlin) [reply skyportal/handlers/api/observation.py]: yeah removed.

File: skyportal/tests/flaky/test_frontend_flaky.py (hunk 2 above)
Comment (mcoughlin) [reply skyportal/tests/flaky/test_frontend_flaky.py]: got this one too.

**Comment by guynir42** [comment]
...and now you have merge conflicts.

**Comment by mcoughlin** [comment]
@guynir42 Please merge if happy.

**Review by guynir42 — APPROVED** [review]

**Pull request merged**